pyproject.toml

```diff
@@ -29,9 +29,9 @@ classifiers = [
 ]
 dependencies = [
     "networkx>=3.0,<=3.3",
-    "phenolrs",
-    "python-arango",
-    "adbnx-adapter"
+    "phenolrs~=0.5",
+    "python-arango~=8.1",
+    "adbnx-adapter~=5.0.5"
```

Comment:
Good, but we should introduce a process to update those from time to time. Also potentially scan for vulnerabilities.

Reply:
Ticket would be nice for this as well - but related to all repositories - not only this one.